frontend/src/components/common/ActionMenuItem/ActionMenuItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { Icon, IconifyIcon, IconProps } from '@iconify/react';
+import { ListItemIcon, ListItemText, MenuItem } from '@mui/material';
+import React from 'react';
+
+export interface ActionMenuItemProps {
+  /** A short description of the action. */
+  description: string;
+  /** Either a string icon, or imported icon. */
+  icon: string | IconifyIcon;
+  /** The action when it's activated. */
+  onClick: React.MouseEventHandler<HTMLLIElement>;
+  /** A longer description of the action. Used in the tooltip. */
+  longDescription?: string;
```

Comment:
This overrides the description in the code, which would surprise me.
I'd prefer to have `description` and `tooltip`. And if tooltip is undefined, we use the description for it.
This way we are flexible. We could even make the description optional (and make the item text component optional) if it makes sense to have menu items with icons only.